python/GangaLHCb/__init__.py

```diff
@@ -82,16 +83,30 @@ def _store_root_version():
 
 def _store_dirac_environment():
     from GangaDirac.Lib.Utilities.DiracUtilities import write_env_cache, get_env
-    diracversion = _guess_version('LHCBDIRAC')
     platform = os.environ['CMTOPT']
+    diracversion = Ganga.Utility.Config.getConfig('LHCb')['diracVersion']
     fdir = os.path.join(os.path.expanduser("~/.cache/Ganga/GangaLHCb"), platform)
-    fname = os.path.join(fdir, diracversion)
+    fname = os.path.join(fdir, defaultLHCbDirac)
```

Comment:
I think it's sensible if that version doesn't exist to exit as something has gone wrong with setting up the environment. I'm not sure we plan to advertise changing this to users as it's mainly a change for us devs